amenity-points.mss

```diff
@@ -2565,10 +2594,17 @@
       [feature = 'leisure_ice_rink'],
       [feature = 'leisure_pitch'] {
         text-fill: darken(@pitch, 40%);
+        [sport = 'baseball'],
+        [sport = 'basketball'],
+        [sport = 'baseball'],
+        [sport = 'soccer'],
+        [sport = 'tennis'] {
+		  text-dy: 10;
+        }
       }
     }
   }
-
+  
```

Comment:
Please remove trailing spaces.

Reply:
I did what @jeisenbe suggested and it didn't work. Maybe you or him can look over it and tell me what I did wrong. SQL isn't really my area of expertise.

Reply:
Not to be a jerk or anything, but does it matter what at all what I feel doing with a PR or not? I was fine with it being rejected if the code was to complicated. Now its extra work that I didn't want to do and @jeisenbe could have done himself in another PR. Since he's the one that suggested it and knows how to do that stuff. It does zero good though if I screw the PR up in the process of trying to implement something I don't have the knowledge how to do. Or I probably would have just done it that way in the first place. I'm lucky I got the icons to render as it was.